docs/en/admins/03_Updating.md

```diff
@@ -0,0 +1,92 @@
+
+First things first: we recommend to create a backup before updating:
+
+```sh
+# Perform all commands below in your FreshRSS directory:
+cd /usr/share/FreshRSS
+
+tar -czvf FreshRSS-backup.tgz .
+```
+
+The update process depends on your installation type, see below:
+
+
+## Using the web admin panel
+
+Change to your installation at http://localhost/FreshRSS/p/i/?c=update and hit the "Check for new updates" button.
+
+If there is a new version you will be prompted again. 
+
+
+## Using git
+
+If you manage FreshRSS via command line, then installing and updating FreshRSS can be done via git:
+
+```sh
+# If your local user does not have write access, prefix all commands by sudo:
+sudo ...
+
+# Perform all commands below in your FreshRSS directory:
+cd /usr/share/FreshRSS
+
+# Use the development version of FreshRSS
+git checkout -b dev origin/dev
+
+# Check out a specific version of FreshRSS
+# See release names on https://github.com/FreshRSS/FreshRSS/releases
+# You will then need to manually change version
+# or checkout master or dev branch to get new versions
+git checkout 1.7.0
+
+# Verify what branch is used
+git branch
+
+# Check whether there is a new version of FreshRSS,
+# assuming you are on the /master or /dev branch
+git fetch --all
+git status
+
+# Discard manual changes (do a backup before)
+git reset --hard
+# Then re-delete the file forcing the setup wizard
+rm data/do-install.txt
+
+# Delete manual additions (do a backup before)
+git clean -f -d
+
+# Update to a newer version of FreshRSS,
+# assuming you are on the /master or /dev branch
+git pull
+
+# Set the rights so that your Web server can access the files
+# (Example for Debian / Ubuntu)
+chown -R :www-data . && chmod -R g+r . && chmod -R g+w ./data/
+```
+
+
+## Using the zip archive
+
+If you installed FreshRSS with the ZIP archive you have to perform a couple more steps:
+
+```sh
+# If your local user does not have write access, prefix all commands by sudo:
+sudo ...
```

Comment:
Wouldn't it look better/be more readable to only do that to the commands, like this.

<hr>

If your local user does not have write access, prefix all commands by sudo:
```
sudo ...
```

Reply:
I'll push one chapter with that formatting and then you can have a look what you like more

Reply:
Just to be explicit about what I mean by "that formatting." ;-)

### Commands intended to be executed in order (you can c/p the whole block if desired)
```sh
# creates directory
mkdir bla
# change to said directory
cd bla
```

### Random commands
Creates directory:
```sh
mkdir bla
```
Creates file:
```sh
touch bla
```

Reply:
PS Of course those two can follow each other. So we have one block above, followed by two separate blocks about file permissions (first-time only?) and sudo.

Reply:
Just to make sure: shall I revert everything back to the former state? From my opinion the previous was cleaner

Reply:
I'd personally separate out the sudo thing but either way.

Reply:
I will remove that sudo completely. It was part of the already existing documentation, but I doubt that someone has a server and installs FreshRSS without basic knowledge of the bash and permissions.